### PR TITLE
Add "text to speech bubble" feature w/ Agent J and Jibhat pictures.

### PR DESCRIPTION
**Pull request by fractalpenguin**

### CODE REVIEW

File: bot/constants.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
+"""
+A list of constants.
+"""
 # Fun constants
 
 EVERYONE_REACTIONS = ["🙁", "🇼", "🇭", "🇴", "🔵", "🇵", "🇮", "🇳", "🇬", "🇪", "🇩"]
-QUOTE_CHANNEL_ID = 463657120441696256
```
Comment (thebeanogamer) [inline bot/constants.py]: This is the wrong id

File: requirements.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-https://github.com/Rapptz/discord.py/archive/rewrite.zip#egg=discord.py[voice]
\ No newline at end of file
+git+https://github.com/Rapptz/discord.py.git@rewrite#egg=discord.py[voice]
```
Comment (thebeanogamer) [inline requirements.txt]: Why are you changing this?

File: bot/cogs/fun.py [hunk 3]
```diff
@@ -99,11 +101,12 @@ def __init__(self, bot: Bot):
         if unknown_emojis:
             emoji_string = ", ".join(unknown_emojis)
             # Inserts an invisible character to render mass mentions ineffective.
-            emoji_string = emoji_string.replace("@here", "@\xadhere").replace("@everyone", "@\xadeveryone")
+            emoji_string = emoji_string.replace(
+                "@here", "@\xadhere").replace("@everyone", "@\xadeveryone")
```
Comment (thebeanogamer) [inline bot/cogs/fun.py]: Why are you changing this?

**Comment by thebeanogamer** [comment]
It may also be worth adding a note to the readme.md to explain the new commands.

**Comment by thebeanogamer** [comment]
Codacy seems to be hanging. If Travis passess, but Codacy is stuck, I'm fine with merging.

File: requirements.txt (hunk 2 above)
Comment (fractalpenguin) [reply requirements.txt]: It's the preferred way of referring to a git repository in the requirements.txt file:
https://pip.readthedocs.io/en/stable/reference/pip_install/#git

File: bot/cogs/fun.py (hunk 3 above)
Comment (fractalpenguin) [reply bot/cogs/fun.py]: That was due to the line being over 80 characters, which goes against PEP8:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

**Pull request merged**